src/include/zf/zf_stack.h

```diff
@@ -167,5 +167,81 @@ enum zf_stack_feature {
 */
 ZF_LIBENTRY int zf_stack_query_feature(struct zf_stack* stack, enum zf_stack_feature feature);
 
+/*! \brief Layout for a field of statistics */
+typedef struct {
+  /** Name of statistics field */
+  char* evsfl_name;
+  /** Offset of statistics file, in bytes */
+  int   evsfl_offset;
+  /** Size of statistics field, in bytes */
+  int   evsfl_size;
+} zf_stats_field_layout;
+
+/*! \brief Layout for statistics */
+typedef struct {
+  /** Size of data for statistics */
+  int                      evsl_data_size;
+  /** Number of fields of statistics */
+  int                      evsl_fields_num;
+  /** Array of fields of statistics */
+  zf_stats_field_layout    evsl_fields[];
+} zf_stats_layout;
+
+/*! \brief  */
+typedef struct {
+  /** Number of underlying interfaces present in stack */
+  int               num_intfs;
+  /** Array of layouts, one per interface */
+  zf_stats_layout** layout;
+} zf_stats_collection;
```

Comment:
Think this should have layout in name as doesn't contains stats.
could perhaps just call this zf_stats_layout and name the data structures above zf_nic_stats_...

Reply:
The `zf_stats_field_layout` and `zf_stats_layout` relate directly to the `ef_vi_...` ones of the same name. I've updated this to `zf_layout_collection`, since that is what it's mostly used for.